src/help/help.cpp

```diff
@@ -165,13 +165,17 @@ void show_help(CVideo& video, const section &toplevel_sec,
 	CVideo& screen = video;
 	const surface& scr = screen.getSurface();
 
+#ifndef __IPHONEOS__
 	const int width  = std::min<int>(font::relative_size(1200), scr->w - font::relative_size(20));
 	const int height = std::min<int>(font::relative_size(850), scr->h - font::relative_size(150));
+#else
+	const int width  = scr->w;
+	const int height = scr->h - font::relative_size(70);
+#endif
```

Comment:
Could be helpful to also implement this in the unfinished GUI2 version of help, which I think is already present in the repository; might save some extra work once it's actually finished.